README.md

```diff
@@ -195,6 +195,26 @@ var stringElement = minim.toElement("foobar");
 var stringElementClone = stringElement.clone();
 ```
 
+#### findRecursive
+
+Recursively find an element. Returns an ArrayElement containing all elements
+that match the given element name.
+
+```javascript
+const strings = element.findRecursive('string');
+```
+
+You can pass multiple element names to `findRecursive`. The method will
+search for the last given element name inside elements matching the other
+given element names.
+
+```javascript
+const stringInsideMembers = element.findRecursive('member', 'string');
+```
+
+**NOTE:** *The returned elements contain a `parents` property which includes
+all of the elements parent in the original Refract tree.*
```

Comment:
We need to give an example for this `parents` property. I should have expected it but I was surprised to find that the order is from top to bottom.